samples/Face/Verify.cs

```diff
@@ -32,14 +33,16 @@ public static async Task Run(string endpoint, string key)
             List<DetectedFace> detectedFaces1 = await Common.DetectFaces(
                                                     client,
                                                     $"{ImageUrlPrefix}{sourceImageFileName1}",
-                                                    recognitionModel: recognitionModel);
```

Comment:
We should also add `detectionModel: detectionModel` on the 27th line's `Common.DetectFaces`